api/src/main/java/org/openmrs/module/ugandaemrfingerprint/core/FingerPrintConstant.java

```diff
@@ -28,14 +28,23 @@
     public static final String PATIENT_ID = "patient.id";
 
     public static final String PATIENT_UUID_SEARCH_STRING = "{patient(patients:[\"%s\"]){uuid birthdate gender dead patientFacility{ uuid name} names{ familyName middleName givenName voided} summaryPage{obs {uuid concept encounterDate encounterType valueCoded valueText valueNumeric valueDatetime valueDrug valueBoolean valueComplex voided }} mostRecentEncounter{obs { uuid concept encounterDate encounterType valueCoded valueText valueNumeric valueDatetime valueDrug valueBoolean valueComplex voided}}}}";
-    public static final String PATIENT_NATIONAL_ID_SEARCH_STRING = "{patient(identifier:{t:\"f0c16a6d-dc5f-4118-a803-616d0075d282\",v:\"%s\"}){uuid birthdate gender dead patientFacility{ uuid name} names{ familyName middleName givenName voided}}}";
+    public static final String PATIENT_NATIONAL_ID_SEARCH_STRING = "{patient(identifier:{t:\"f0c16a6d-dc5f-4118-a803-616d0075d282\",v:\"%s\"}){uuid birthdate gender dead patientFacility{ uuid name} names{ familyName middleName givenName voided} summaryPage{obs {uuid concept encounterDate encounterType valueCoded valueText valueNumeric valueDatetime valueDrug valueBoolean valueComplex voided }} mostRecentEncounter{obs { uuid concept encounterDate encounterType valueCoded valueText valueNumeric valueDatetime valueDrug valueBoolean valueComplex voided}}}}";
+    public static final String PATIENT_NATIONAL_ID_SIMPLE_SEARCH_STRING = "{patient(identifier: {t:\"f0c16a6d-dc5f-4118-a803-616d0075d282\", v:\"%s\"}) {uuid}}";
+    public static final String PATIENT_DETAILS = "{patient(patients: [\"%s\"]) {uuid birthdate gender dead patientFacility {uuid name} names {familyName middleName givenName voided}}}";
+    public static final String MOST_RECENT_ENCOUNTER = "{patient(patients: [\"%s\"]) {uuid birthdate gender dead patientFacility {uuid name} names {familyName middleName givenName voided} mostRecentEncounter {obs {uuid concept encounterDate encounterType valueCoded valueText valueNumeric valueDatetime valueDrug valueBoolean valueComplex voided}}}}";
+    public static final String SUMMARY_ENCOUNTER = "{patient(patients: [\"%s\"]) {uuid birthdate gender dead patientFacility {uuid name} names {familyName middleName givenName voided} summaryPage {obs {uuid concept encounterDate encounterType valueCoded valueText valueNumeric valueDatetime valueDrug valueBoolean valueComplex voided}}}}";
     public static final String PATIENT_ONLINE_ID = "identifiers";
     public static final String PATIENT_NOT_FOUND = "Patient Not Found";
     public static final String CONNECTION_TEST_IP = "google.com/";
     public static final String DEVICE_SOCKET_IP = "ugandaemrfingerprint.socketIPAddress";
```

Comment:
Consider adding a prefix to Global properties to make them stand out 

```suggestion
    public static final String GP_DEVICE_SOCKET_IP = "ugandaemrfingerprint.socketIPAddress";
```

Reply:
Noted